src/emcc_wrapper.sh

```diff
@@ -4,7 +4,16 @@ SCRIPT_FILE=$(readlink -f "${BASH_SOURCE}")
 SCRIPT_DIR=$(cd $(dirname "${SCRIPT_FILE}") && pwd)
 WASM_ROOT=$(dirname "${SCRIPT_DIR}")
 
+# Without this emscripten will attempt to write an '_santiy' config
```

Comment:
s/santy/sanity and maybe 'a' instead of 'an' depending on whether you pronounce uderscores? :)